.github/workflows/ci-test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-java@v4
         with:
-          distribution: temurin
+          distribution: liberica
```

Comment:
How does a different Java implementation impact the journal recovery output?

Reply:
Moved JDK change to PR - https://github.com/eXist-db/exist/pull/5480